sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/ReleaseDownloadListener.java

```diff
@@ -140,8 +139,13 @@ public boolean onComplete(@NonNull Uri localUri) {
 
     @Override
     public void onError(@NonNull String errorMessage) {
+
+        /*
+         * TODO: Add a generic error message to resources (with translations) to show the toast.
```

Comment:
We'll start with generic messages in english in resources. We can translate later with the help of a team.